src/libsyncengine/requests/serverrequests.cpp

```diff
@@ -1067,63 +1067,41 @@ ExitCode ServerRequests::getPublicLinkUrl(int driveDbId, const QString &fileId,
         return ExitCode::DataError;
     }
 
-    if (auto exitCode = job->runSynchronously(); exitCode != ExitCode::Ok) {
+    if (job->runSynchronously() != ExitCode::Ok) {
         if (job->exitCode() == ExitCode::BackError && job->exitCause() == ExitCause::ShareLinkAlreadyExist) {
```

Comment:
```suggestion
        if (job->exitCode() == ExitCode::BackError && job->exitCause() == ExitCause::ShareLinkAlreadyExists) {
```

If it does not cause too many replacements.